### ISSUE TITLE
Details of the Experiment on Replica Dataset for Reproducing Results in Paper

### ISSUE THREAD

**Issue by OceanYing** [Jumpat/SegmentAnythingin3D#37]
Hi, thanks for your impressive and effective work!
I wonder would you like to provide more details about the evaluation on Replica Dataset? Specifically, I have the following questions:

1. You choose one reference view for each object, would you please release the reference view ID for us to reproduce the results?
2. For training, did you use both sequence 1 and 2 in the Replica dataset or sequence 1 only? From your dataloader, the step size is set as 5. What about the step size for evaluation?
3. There are many objects in the dataset that occupy only a small fraction of the pixels in some images, which may cause low IoU results. Did you use some filter scheme to suppress this problem? (eg. if pixnum($obj_i$ in $img_j$) < 100, then exclude the $img_j$ from the evaluation procedure of $obj_i$)

**Comment by Jumpat** [comment]
Hello! Thanks for your interest! 
The reference view for each object is selected as the one with the most pixels occupied by it. For example, if an object A appears in view 1,3,5,7, and occupy 10, 5, 7 30 pixels respectively, we choose view 7 as the reference view since it has the most pixels occupied by object A.
For training and testing, we only use the sequence 1 only. Strictly speaking, there is no training phase in SA3D since no ground-truth is used. The evaluation is performed just on the same set of images of the "training".
In fact, we found Replica is not a good dataset for evaluating the interactive 3D segmentation, which has a lot of strange annotations (we explained this problem in the appendix of our paper). We provide a filtered object id list for each scene in the Replica.
